### PR TITLE
update id web after releases

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

**Comment by jmprieur** [comment]
Let's also update MicrosoftIdentityWebVersion to 3.2.1 for next time.

**Review by jmprieur — APPROVED** [review]
LGTM
Let's also update MicrosoftIdentityWebVersion to 3.2.1

**Comment by jennyf19** [comment]
> LGTM Let's also update MicrosoftIdentityWebVersion to 3.2.1

dependabot will provide PRs, but I'll do this later

File: src/Microsoft.Identity.Web.TokenAcquisition/Microsoft.Identity.Web.TokenAcquisition.csproj [hunk 1]
```diff
@@ -46,7 +46,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' Or '$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.2.0" />
```
Comment (jennyf19) [inline src/Microsoft.Identity.Web.TokenAcquisition/Microsoft.Identity.Web.TokenAcquisition.csproj:49]:
```suggestion
    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.0" />
```

**Pull request merged**